doc/pprof.md

```diff
@@ -235,10 +239,41 @@ them. This is useful to combine profiles from multiple processes of a
 distributed job. The profiles may be from different programs but must be
 compatible (for example, CPU profiles cannot be combined with heap profiles).
 
-pprof can subtract a profile from another in order to compare them. For that,
-use the **-base= _profile_** option, where *profile* is the filename or URL for the
-profile to be subtracted. This may result on some report entries having negative
-values.
+
+## Comparing profiles
+
+pprof can subtract a profile from another, provided the profiles are of 
+compatible types, in order to compare them. For that, use the 
+**-diff_base= _profile_** option, where *profile* is the filename or URL for the
+profile to be subtracted. This may result in some report entries having negative
+values.  Percentages in the report are relative to the total of 
+the diff base. 
+
+If the merged profile is output as a protocol buffer, all samples in
+the diff base profile will have a label with the key "pprof::base" and a value
+of "true". If pprof is then used to look at the merged profile, it will behave
+as if separate source and base profiles were passed in.
+
+The **-base** option can be used in the place of the **-diff_base** option, and
+may be preferrable when comparing two cumulative profiles, for example two
+contention profiles collected from the same program at different times. When 
```

Comment:
This should be more specific that we are talking about contention reports from Go programs. Linking to Go's documentation would be good.

Reply:
I can't find much in go documentation on contention reports.

This is the main thing I found (from [here](https://golang.org/src/runtime/pprof/pprof.go?h=block+profile)):
```
//	goroutine    - stack traces of all current goroutines
//	heap         - a sampling of all heap allocations
//	threadcreate - stack traces that led to the creation of new OS threads
//	block        - stack traces that led to blocking on synchronization primitives
//	mutex        - stack traces of holders of contended mutexes
```

I'm guessing that block profiles are what we would consider contention profiles?
I could look for blog posts?

Reply:
Maybe link to https://golang.org/doc/diagnostics.html#profiling? It's not exclusively about contention but has something.